cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -436,6 +458,61 @@ func parseCPUShare() (int64, error) {
 	return nano.Num().Int64(), nil
 }
 
+func getDisks() ([]*provision.Disk, error) {
+	// should have at least a single primary disk
+	disks := []*provision.Disk{
+		{
+			Size: int64(clusterDiskSize) * 1024 * 1024,
+		},
+	}
+
+	for _, disk := range clusterDisks {
+		var (
+			partitions     = strings.Split(disk, ",")
+			diskPartitions = make([]*v1alpha1.DiskPartition, len(partitions))
+			diskSize       int64
+		)
+
+		for j, partition := range partitions {
+			parts := strings.Split(partition, ":")
+			if len(parts) != 2 {
+				return nil, fmt.Errorf("failed to parse malformed partition definition %s", partition)
+			}
+
+			partitionPath := parts[0]
+
+			if !strings.HasPrefix(partitionPath, "/") {
+				return nil, fmt.Errorf("partition path should be absolute")
+			}
+
+			partitionSize, e := strconv.ParseInt(parts[1], 10, -1)
+
+			if e != nil {
+				s, e := humanize.ParseBytes(parts[1])
```

Comment:
Just double checking, this is backwards compatible?

Reply:
These arguments are new, so shouldn't have any backward compatibility issues.